filebeat/autodiscover/builder/hints/logs.go

```diff
@@ -186,7 +186,7 @@ func (l *logHints) getFilesets(hints common.MapStr, module string) map[string]*f
 
 	moduleFilesets, err := l.Registry.ModuleFilesets(module)
 	if err != nil {
-		logp.Err("Error retrieving module filesets", err)
+		logp.Err("Error retrieving module filesets: %s", err)
```

Comment:
use %+v

Reply:
On my IDE, it's showing `logp.Err` is deprecated. Should we use `logp.Error` instead?

Reply:
@ph Changed it but can you share why you want to use `%+v`?

@kaiyan-sheng Yes, that is a general thing we should do across the code base. But changing to `logp.Error` requires more change and this is just a quick fix to to the existing message.

Reply:
> the value in a default format 	when printing structs, the plus flag (%+v) adds field names.

It's just usually a better way to do it if you use compound errors.